lib/puptime/service/tcp.rb

```diff
@@ -24,18 +23,12 @@ def initialize(name, id, type, interval, options = {})
         @tcp_service = parse_tcp_params(options)
       end
 
-      def run
-        @scheduler_job_id = @scheduler.every @interval, overlap: false, job: true do
-          ping
-        end
-      end
-
-      def ping
+      def self.ping
```

Comment:
I don't understand why this has to be a class method. Likewise for other ping functions